contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,227 @@
+            "@great_expectations",
+        ],
+        "requirements": [
+            "sqlalchemy",
+            "snowflake-sqlalchemy",
+            "snowflake-connector-python",
+        ],
+        "has_full_test_suite": False,
+        "manually_reviewed_code": True,
+    }
+
+    metric_dependencies = ("query.template_values",)
+    template_dict: dict
+    query = """
+        SELECT count(1) FROM (
+        SELECT a.{foreign_key_column}
+        FROM {active_batch} a
+        LEFT JOIN {foreign_table} b
+            ON a.{foreign_key_column} = b.{primary_key_column_in_foreign_table}
+        WHERE b.{primary_key_column_in_foreign_table} is NULL)
+        """
+    success_keys = (
+        "template_dict",
+        "query",
+    )
+    domain_keys = ("query", "batch_id", "row_condition", "condition_parser")
+
+    default_kwarg_values = {
+        "catch_exceptions": False,
+        "meta": None,
+        "query": query,
+    }
+
+    @classmethod
+    @override
+    @renderer(renderer_type="renderer.prescriptive")
+    def _prescriptive_renderer(
+        cls,
+        configuration: Optional[ExpectationConfiguration] = None,
+        result: Optional[ExpectationValidationResult] = None,
+        runtime_configuration: Optional[dict] = None,
+    ) -> List[RenderedStringTemplateContent]:
+        runtime_configuration = runtime_configuration or {}
+        styling = runtime_configuration.get("styling")
+
+        template_dict = configuration.kwargs.get("template_dict")
+
+        template_str = "No values in the column $foreign_key_column are expected to be missing in table $foreign_table 's $primary_key_column_in_foreign_table column."
+
+        params = {
+            "foreign_key_column": template_dict["foreign_key_column"],
+            "foreign_table": template_dict["foreign_table"],
+            "primary_key_column_in_foreign_table": template_dict[
+                "primary_key_column_in_foreign_table"
+            ],
+        }
+
+        return [
+            RenderedStringTemplateContent(
+                content_block_type="string_template",
+                string_template={
+                    "template": template_str,
+                    "params": params,
+                    "styling": styling,
+                },
+            )
+        ]
+
+    def _validate_template_dict(
+        self, configuration: Optional[ExpectationConfiguration] = None
+    ) -> None:
+        template_dict = configuration.kwargs.get("template_dict")
+        if not isinstance(template_dict, dict):
+            raise TypeError("template_dict must be supplied as a dict")
+        if not all(
+            [
+                "foreign_key_column" in template_dict,
+                "foreign_table" in template_dict,
+                "primary_key_column_in_foreign_table" in template_dict,
+            ]
+        ):
+            raise KeyError(
+                "The following keys have to be in the template dict: id, foreign_table, foreign_key_column"
```

Comment:
Should `id` in this error message be `primary_key_column_in_foreign_table` instead?

Also, nit - as a user, I would find it more helpful to know what is missing...e.g.,


```
        if not all(
            [
                "foreign_key_column" in template_dict,
                "foreign_table" in template_dict,
                "primary_key_column_in_foreign_table" in template_dict,
            ]
        ):
            raise KeyError(
                f"The following keys are missing from the template dict: 
{'foreign_key_column ' if  not ('foreign_key_column' in template_dict) else ''}
{'foreign_table ' if  not ('foreign_table' in template_dict) else ''}
{'primary_key_column_in_foreign_table ' if  not ('primary_key_column_in_foreign_table' in template_dict) else ''}"
```

(there is prob a better way to write that and make it readable but that is the first thing that came to mind)

Reply:
This is a great suggestion. Thank you so much @JennyTee